.github/fabricbot.json

```diff
@@ -707,7 +707,7 @@
         "deleteBranches": true,
         "allowAutoMergeInstructionsWithoutLabel": true,
         "enforceDMPAsStatus": true,
-        "removeLabelOnPush": true,
+        "removeLabelOnPush": false,
```

Comment:
For the automation pr, `msftbot` was removing the label as `github-actions` does not have write access for next.